prepline_general/api/app.py

```diff
@@ -31,6 +32,9 @@
     openapi_tags=[{"name": "general"}],
 )
 
+metrics_app = make_asgi_app()
+app.mount("/metrics", metrics_app)
```

Comment:
Pointing out that `/metrics` calls are not being logged already:
https://github.com/Unstructured-IO/unstructured-api/blob/b43fef0a82080ff9cf40cf126c7e17ce3b36a61c/prepline_general/api/app.py#L121-L124